docs-md/getting-started/index.md

```diff
@@ -27,7 +27,7 @@ make sure you update CocoaPods using `pod repo update` before starting a new pro
 
 ## Adding Capacitor to an existing web app
 
-Capacitor was designed to drop into any existing modern JavaScript web app. A valid `package.json` file and a folder containing all web assets are required to get started.
+Capacitor was designed to drop into any existing modern JavaScript web app. A valid `package.json` file and a folder containing all web assets are required to get started. Additionally, ensure that a `<head>` element is available in the main `index.html` file, as Capacitor is injected there upon app initialization.
```

Comment:
Edit accepted, thanks. I am assuming we would show up as co-authors in the merge. I did take the liberty to update the title as well.

Another idea, wonder if it makes sense or not:

```suggestion
Capacitor was designed to drop into any existing modern JavaScript web app. A valid `package.json` file and a folder containing all web assets are required to get started. In addition, a `<head>` element is needed in the main `index.html` file, as Capacitor is injected there upon app initialization.
```